server/src/queries/mod.rs

```diff
@@ -39,5 +88,49 @@ pub async fn get_signals(db: &Pool<Postgres>) -> Result<Vec<DBResponse>, sqlx::E
         AND NOT b.chain_status = 'orphaned'
         AND buc.status = 'applied'
         "#
-    ).fetch_all(db).await
+        ).fetch_all(db).await
+    }
+}
+
+pub async fn build_router(context: ApiContext) -> Router {
+    let cors = CorsLayer::new()
+        .allow_methods([Method::GET, Method::POST])
+        .allow_origin(Any);
+    axum::Router::build_v1(&context.config)
+        .layer(ServiceBuilder::new().layer(cors).layer(Extension(context)))
+}
```

Comment:
This function should be defined in `router/mod.rs` - it's a bit confusing having this in the queries module.